paper.tex

```diff
@@ -526,6 +526,32 @@ \subsubsection{Messages Publishment}
 
 Therefore, transferring MAM operations to brokers while ensuring the profit and privacy of providers through blind signatures can effectively solve performance problems and lower the threshold to participate in such framework. Brokers can be PCs or powerful machines that runs Ethereum client and Tangle-accelerator\cite{TA}. Where Ethereum client is used to interact with Ethereum and Tangle-accelerator is a caching proxy server for IOTA, which can serve thousands of IOTA requests at once without accessing remote IOTA full nodes frequently and provide PoW acceleration. However, MAM operations still cost a considerable time that improving the performance of MAM is an essential issue that needs to be done for the next step.
 
+\subsection{MAM versus E2EE}
```

Comment:
Isn't this section should be in section Evaluation?

Reply:
Under MAM eveluation?

Reply:
It's ok to open a new subsection.

Reply:
Ok

Reply:
The comparison should be "MAM vs. the delegated MAM."